test/multiverse/suites/active_record_pg/Envfile

```diff
@@ -28,7 +28,6 @@ def gem_list(activerecord_version = nil)
 end
 
 create_gemfiles(ACTIVERECORD_VERSIONS, gem_list)
-
```

Comment:
This wasn't really a line removal as much as it was adding our license & heading to the top of this file